### PR TITLE
Update minimal mode for new Google formatting

### LINKED ISSUE #634: [BUG] `WHOOGLE_MINIMAL` removes search results
**Describe the bug**
After some time, in some cases, a Whoogle search leads to an empty search page.

**To Reproduce**
1. Search (though it's unreliable)
2. See blank page

**Deployment Method**
- [ ] Heroku (one-click deploy)
- [ ] Docker
- [x] `run` executable
- [ ] pip/pipx
- [ ] Other: [describe setup]

**Version of Whoogle Search**
- [x] Latest build from [source] (i.e. GitHub, Docker Hub, pip, etc)
- [ ] Version [version number]
- [ ] Not sure

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser: firefox

**Smartphone (please complete the following information):**

**Additional context**
image

### PR DESCRIPTION
**Pull request by DUOLabs333**
Fixes #634

### CODE REVIEW

**Comment by DUOLabs333** [comment]
Am I good to go?

File: app/filter.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from urllib.parse import parse_qs
 import os
 
+minimal_mode_sections = ['Top stories','Images','People also ask']
```
Comment (benbusby) [inline app/filter.py:15]: Pep8 failure here:

```suggestion
minimal_mode_sections = ['Top stories', 'Images', 'People also ask']
```

**Review by benbusby — CHANGES_REQUESTED** [review]
Looks fine aside from the pep8 issue (you should get notifications when that check fails for the PR).

I'm not crazy about this solution only working for English language users, but I'm fine with merging for now since it's fairly minimal and don't have a good suggestion immediately in mind for making this work for all languages.

**Comment by DUOLabs333** [comment]
Oh, I didn't even realize that. You're right, I don't know how else it would be done.

**Comment by DUOLabs333** [comment]
Ok, finished.

**Review by benbusby — APPROVED** [review]

**Pull request merged**